packages/lib/slots.ts

```diff
@@ -208,11 +208,22 @@ const getSlots = ({
   });
 
   if (!!activeOverrides.length) {
-    const overrides = activeOverrides.flatMap((override) => ({
-      userIds: override.userId ? [override.userId] : [],
-      startTime: override.start.getUTCHours() * 60 + override.start.getUTCMinutes(),
-      endTime: override.end.getUTCHours() * 60 + override.end.getUTCMinutes(),
-    }));
+    const overrides = activeOverrides.flatMap((override) => {
+      const organizerUtcOffset = dayjs(override.start.toString()).tz(organizerTimeZone).utcOffset();
+      const inviteeUtcOffset = dayjs(override.start.toString()).tz(timeZone).utcOffset();
```

Comment:
NIT
```suggestion
      const organizerUtcOffset = dayjs(override.start).tz(organizerTimeZone).utcOffset();
      const inviteeUtcOffset = dayjs(override.start).tz(timeZone).utcOffset();
```

Not tested but do we need this to be toString? We seem to just use override.start/end everywhere

Reply:
toString and non-toString are basically identical. for Dayjs purposes I'd probably do, but it's probably all the same:

`Dayjs.utc(override.start.toISOString()).tz(...`